test/LEGO.AsyncAPI.Tests/TestData/AsyncApiSchema_InlinedReferences.yml

```diff
@@ -0,0 +1,27 @@
+asyncapi: 2.6.0
```

Comment:
I am not sure how you feel about this, but I find some of the unit tests very very hard to read. Having such large data blobs just looks terrible to me. So I attempted to move the biggest culprits out into their own file. Ideally some of these tests could be more granular instead of just testing everything

Reply:
Yeah, a lot of them were built out over time as bugs emerged, and were generally added to existing document tests.
But ive tried to define them either as 'full' document tests or fragment tests.
As we can deserialize fragments directly.

The biggest culprit for me, is the duplication of documents in the tests where we want to check that the serialized and deserialized match, so to speak. So we define both the object model AND a yml spec.

But generally, a nice change (y)